File: bin/com/problem/cpp/codechef/CourseRegistration.py
T = int(input())
def solve(n,m,k):
    courseLeft = m - k
    if n > courseLeft: print("No")
     if n <= courseLeft: print("Yes"); 

for _ in range(T):
    n,m,k = map(int, input().split())
    solve(n,m,k)
    

File: bin/com/problem/cpp/codechef/CourseRegistration.cpp
#include <iostream>
using namespace std;

void solve(int n, int m, int k) {
    int courseLeft = m - k;
    if (n > courseLeft) cout << "No" << endl;
    else if (n <= courseLeft) cout << "Yes" << endl;
    else cout << "Yes" << endl;
}

int main() {
    int k = (cin >> k, k);
    for (int i = 1; i <= k; i++) {
        int n, m, k = (cin >> n >> m >> k,n,m,k);   
        solve(n, m, k);
    }
}